llvm-spirv/lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -840,7 +840,9 @@ SPIRVFunction *LLVMToSPIRVBase::transFunctionDecl(Function *F) {
     if (Attrs.hasParamAttr(ArgNo, Attribute::ReadOnly))
       BA->addAttr(FunctionParameterAttributeNoWrite);
     if (Attrs.hasParamAttr(ArgNo, Attribute::ReadNone))
-      BA->addAttr(FunctionParameterAttributeNoReadWrite);
+      // TODO: intel/llvm customization
```

Comment:
Please add a link to the intel/llvm issue here in the comment.

Reply:
done